### DISCUSSION TITLE
connection migrate功能大概什么时候能够支持？

### DISCUSSION THREAD

**Discussion by sharewzy** [Tencent/tquic#339] [Ideas（新功能的建议和讨论）]
connection migrate功能大概什么时候能支持呢？看到代码里面有接口存在，但是没有协议实现部分。

**Comment by iyangsj** [comment]
@sharewzy 目前TQUIC服务端实现支持处理连接迁移；客户端实现不支持主动发起连接迁移

**Reply by iyangsj** [reply]
后续几个版本会重点支持多路径协议的演进，客户端主动发起迁移功能暂无明确发布计划

**Reply by cmsly** [reply]
目前在Android上测试，4g切换wifi后，连接好像正常工作，wifi且换成4g，连接好像是断掉了，客户端网络变化没有处理连接迁移吗？

**Reply by cmsly** [reply]
@iyangsj  目前在Android上测试，4g切换wifi后，连接好像正常工作，wifi且换成4g，连接好像是断掉了，客户端网络变化没有处理连接迁移吗？

**Reply by iyangsj** [reply]
@cmsly 目前不支持，预计会在2024H2 支持

**Reply by cmsly** [reply]
@iyangsj 多谢
